.github/workflows/tests-labs.yml

```diff
@@ -0,0 +1,69 @@
+name: PennyLane Labs Unit-Tests
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
What if a PR is pointing to the RC branch, which also has branch protection? We shouldn't be making PRs to the RC branch for experimental features, but I don't want to assume that it will never happen. cc @mlxd

Reply:
Since labs isn't part of releases yet, we don't need to worry about RC just yet.